src/pages/index.js

```diff
@@ -10,41 +18,54 @@ const headerLogo = document.getElementById("header__logo");
 headerLogo.src = logo;
 
 const profileAvatar = document.getElementById("profile__avatar");
-profileAvatar.src = avatar;
+let selectedCard;
+let selectedCardId;
 
-
-const initialCards = [
-  {
-    name: "Val Thorens",
-    link: "https://practicum-content.s3.us-west-1.amazonaws.com/software-engineer/spots/1-photo-by-moritz-feldmann-from-pexels.jpg",
-  },
-  {
-    name: "Restaurant terrace",
-    link: "https://practicum-content.s3.us-west-1.amazonaws.com/software-engineer/spots/2-photo-by-ceiline-from-pexels.jpg",
-  },
-  {
-    name: "An outdoor cafe",
-    link: "https://practicum-content.s3.us-west-1.amazonaws.com/software-engineer/spots/3-photo-by-tubanur-dogan-from-pexels.jpg",
-  },
-  {
-    name: "A very long bridge, over the forest and through the trees",
-    link: "https://practicum-content.s3.us-west-1.amazonaws.com/software-engineer/spots/4-photo-by-maurice-laschet-from-pexels.jpg",
-  },
-  {
-    name: "Tunnel with morning light",
-    link: "https://practicum-content.s3.us-west-1.amazonaws.com/software-engineer/spots/5-photo-by-van-anh-nguyen-from-pexels.jpg",
+const api = new Api({
+  baseUrl: "https://around-api.en.tripleten-services.com/v1",
+  headers: {
+    authorization: "138fe482-aa90-46db-b20b-613bcdba9e47",
+    "Content-Type": "application/json",
   },
-  {
-    name: "Mountain house",
-    link: "https://practicum-content.s3.us-west-1.amazonaws.com/software-engineer/spots/6-photo-by-moritz-feldmann-from-pexels.jpg",
-  },
-];
+});
+//destructure the second item in the callback of the .then()
+api.getAppInfo().then(([cards, users]) => {
+  console.log(cards);
+  console.log(users);
+  cards.forEach((item) => {
+    const cardElement = getCardElement(item);
+    cardsList.prepend(cardElement);
+  });
+
+  //handle the users information
+  //set the src of avatar img
+  profileAvatar.src = users.avatar;
+  //set textContent of both the text elements
+  document.querySelector(".profile__name").textContent = users.name;
+  document.querySelector(".profile__description").textContent = users.about;
+});
```

Comment:
You need to `catch` possible errors at the very **end** of any server request.

**Please, correct it everywhere in the code**